src/features/common/Layout/UserPropsContext.tsx

```diff
@@ -51,39 +53,37 @@ function UserPropsProvider({ children }: any): ReactElement {
     logout({ returnTo: returnUrl });
   };
 
-  async function loadUser() {
-    setContextLoaded(false);
-    try {
-      const res = await getAccountInfo(token);
-      if (res.status === 200) {
-        const resJson = await res.json();
-        setUser(resJson);
-      } else if (res.status === 303) {
-        // if 303 -> user doesn not exist in db
-        setUser(null);
-        if (typeof window !== 'undefined') {
-          router.push('/complete-signup', undefined, { shallow: true });
+  React.useEffect(() => {
+    async function loadUser() {
+      setContextLoaded(false);
+      try {
+        const res = await getAccountInfo(token, tenantID);
```

Comment:
Try to resolve the TS warning you see on this line. 
Argument of type 'string | undefined' is not assignable to parameter of type 'string'.